bcgov/HOUS-permit-portal
# Submitter version compare

## Description

add updatable permit applications and submitter notifications. 

## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-527

## Steps to QA

see video in card

Log in as submitter
create a permit application
log in as super admin and update the corresponding permit application template
refresh permit application as submitter
see the notification
see the compare box
click links in the compare box
